migrations/1685969609_addSectionTeamGallery.ts

```diff
@@ -0,0 +1,75 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newItemTypes: Record<string, SimpleSchemaTypes.ItemType> = {};
+
+  console.log("Create new models/block models");
+
+  console.log(
+    'Create block model "Section Team Gallery" (`section_team_gallery`)'
+  );
+  newItemTypes["1602503"] = await client.itemTypes.create(
+    {
+      name: "Section Team Gallery",
+      api_key: "section_team_gallery",
+      modular_block: true,
+      hint: "https://www.datocms-assets.com/6524/1685609517-section-team-gallery-preview.png",
+      inverse_relationships_enabled: false,
+    },
+    { skip_menu_item_creation: true }
+  );
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log(
+    'Create Multiple links field "Team" (`team`) in block model "Section Team Gallery" (`section_team_gallery`)'
+  );
+  newFields["8299171"] = await client.fields.create(newItemTypes["1602503"], {
+    label: "Team",
+    field_type: "links",
+    api_key: "team",
```

Comment:
Can we use a label and api_key like `Persons`? I think it's a bit weird to have a singular field name where we're expecting multiple